qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -1066,15 +1071,17 @@ def decomp0(target):
         return U0r, U0l
 
     def decomp1(self, target):
-        r"""
-        Decompose target :math:`~Ud(x, y, z)` with 1 uses of the basis gate :math:`~Ud(a, b, c)`.
-        Result :math:`Ur` has trace:
+        r"""Decompose target :math:`\sim U_d(x, y, z)` with :math:`1` use of the basis gate
+        :math:`\sim U_d(a, b, c)`.
+        Result :math:`U_r` has trace:
 
         .. math::
 
-            |Tr(Ur.U_{target}^{\dag})| = 4|cos(x-a)cos(y-b)cos(z-c) + j sin(x-a)sin(y-b)sin(z-c)|
+            \Big\vert\text{Tr}(U_r \cdot U_\text{target}^{\dag})\Big\vert =
+            4\Big\vert cos(x-a)cos(y-b)cos(z-c) + j sin(x-a)sin(y-b)sin(z-c)\Big\vert
```

Comment:
As above, this is not API-doc'd but if it ever is, the command versions of sin and cos are in normal text font and better distinguishable from variables called "c" "o" and "s" 😄 
```suggestion
            4\Big\vert \cos(x-a)\cos(y-b)\cos(z-c) + j \sin(x-a)\sin(y-b)\sin(z-c)\Big\vert
```